src/components/Reportees/index.jsx

```diff
@@ -1,56 +1,78 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/core/styles';
-import DrilldownIcon from '../DrilldownIcon';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import CONFIG from '../../config';
 
-const styles = ({
+const styles = theme => ({
   root: {
-    margin: '0 0.5rem 0 0',
-  },
-  header: {
-    margin: '0.5rem 0 0.5rem 0',
-  },
-  icon: {
-    fontSize: '1rem',
-    verticalAlign: 'bottom',
-  },
-  person: {
-    // This makes it line up better with the table of components
-    padding: '1px',
-    display: 'flex',
+    width: '50%',
```

Comment:
This doesn't look nice on smaller screens. Use the `Grid` component instead.

![Screen Shot 2019-04-15 at 8 48 15 AM](https://user-images.githubusercontent.com/3766511/56133861-6143b280-5f5b-11e9-9c83-c2bce338ce96.png)

Reply:
I've addressed this:
<img width="417" alt="image" src="https://user-images.githubusercontent.com/44410/56135660-352a3080-5f5f-11e9-9ec1-78219c81fb43.png">